modules/appeals_api/appeals_api_insomnia.json

```diff
@@ -0,0 +1,3797 @@
+{
+  "_type": "export",
+  "__export_format": 4,
+  "__export_date": "2023-04-04T00:04:31.491Z",
+  "__export_source": "insomnia.desktop.app:v2023.1.0",
+  "resources": [
+    {
+      "_id": "req_wrk_d89a54f45e11429abb2af16aa95af1eda7b2e307",
+      "parentId": "fld_wrk_d89a54f45e11429abb2af16aa95af1edb789bc52",
+      "modified": 1679619830033,
+      "created": 1679619830033,
+      "url": "{{ _.base_uri }}/services/appeals/v0/appeals",
+      "name": "List appeals statuses",
+      "description": "",
+      "method": "GET",
+      "body": {},
+      "parameters": [],
+      "headers": [
+        {
+          "name": "X-VA-SSN",
+          "disabled": false,
+          "value": "{{ _.veteran_ssn }}",
+          "id": "pair_134693ecbc944fcbae0fa741a40f9219"
+        },
+        {
+          "name": "X-VA-User",
+          "disabled": false,
+          "value": "{{ _.veteran_email }}",
+          "id": "pair_9d56268d57d24a2dbb91fd5dc87c0b04"
+        }
+      ],
+      "authentication": {
+        "type": "apikey",
+        "key": "apikey",
+        "value": "{{ _.api_key }}",
+        "addTo": "header"
+      },
+      "metaSortKey": -1679619830033,
+      "isPrivate": false,
+      "settingStoreCookies": true,
+      "settingSendCookies": true,
+      "settingDisableRenderRequestBody": false,
+      "settingEncodeUrl": true,
+      "settingRebuildPath": true,
+      "settingFollowRedirects": "global",
+      "_type": "request"
+    },
+    {
+      "_id": "fld_wrk_d89a54f45e11429abb2af16aa95af1edb789bc52",
+      "parentId": "wrk_d89a54f45e11429abb2af16aa95af1ed",
+      "modified": 1679619830035,
+      "created": 1679619830035,
+      "name": "appeals v0",
+      "description": "Caseflow appeals status API",
+      "environment": {
+        "veteran_ssn": "796130115"
```

Comment:
@caseywilliams I noticed a few of the requests included key-value pairs in the "environment" section, but I couldn't find this data in the Insomnia UI. Do you think we should have environment always be empty (e.g. `"environment": {}`) to ensure that the end user always defines the environment variables?

Reply:
I'm making this comment without having any background about Insomnia config, so feel free to tell me why I'm wrong. 🙂

Reply:
Ah yes, nice catch, thanks - it's possible in insomnia to override specific environment variables at the level of the parent folder of a request (you click the dropdown and choose the Environment option) that said though, I don't think we should be doing this - I tried it on a few requests and decided it was too hard to reason about - just missed removing this instance (I'll check for more, too).

I'll refactor this a bit so that (1) there are no more environment-specific settings like this left, and (2) there are a few example SSNs in settings that we can strategically use for `veteran_ssn` in cases where there's no sandbox data available for the default value in the base environment.

Reply:
Gotcha, thanks for the explanation!